src/Employer/Employer.Web/Orchestrators/VacancyManageOrchestrator.cs

```diff
@@ -52,7 +52,7 @@ public async Task<ManageVacancyViewModel> GetManageVacancyViewModel(Vacancy vaca
             viewModel.CanShowEditVacancyLink = vacancy.CanExtendStartAndClosingDates;
             viewModel.CanShowCloseVacancyLink = vacancy.CanClose;
 
-            var vacancyApplicationsTask =  await _client.GetVacancyApplicationsAsync(vacancy.VacancyReference.Value.ToString());
+            var vacancyApplicationsTask =  await _vacancyClient.GetVacancyApplicationsAsync(vacancy.VacancyReference.Value.ToString());
 
```

Comment:
can the names of the variables me more specific , something like, _employerclient and _recruiterclient?

Reply:
I agree however this is the pattern that's been used across the codebase so I'm just fitting in.

Reply:
Do we need IEmployerVacancyClient _client ? As its not being used ?

Reply:
👍 Will remove